packages/chakra-ui/src/components/ui/alert.tsx

```diff
@@ -0,0 +1,33 @@
+import { Alert as ChakraAlert } from '@chakra-ui/react';
+import { CloseButton } from './close-button';
+import * as React from 'react';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { ReactElement, ReactNode, forwardRef } from 'react'`;